tools/render_diff/render_diff.py

```
@@ -0,0 +1,200 @@
+        "unmatched_b": len(b) - matched,
+        "mean_iou": float(np.mean(ious)) if ious else 0.0,
+        "mean_center_px": float(np.mean(centers)) if centers else 0.0,
+    }
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--pdf", required=True)
+    ap.add_argument("--page", type=int, default=0)
+    ap.add_argument("--go-png", default=None, help="PNG rendered by the Go harness (TestRenderDiffGo)")
+    ap.add_argument("--out-dir", default="/tmp/render_diff")
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/render_diff"

(S108)